### ISSUE TITLE
[🐛BUG] Different stopping steps, different results?

### ISSUE THREAD

**Issue by enoche** [RUCAIBox/RecBole#1477]
**Describe the bug**
Best performance can be observed in log file (in training process), with the same results on different stopping steps.
Final results are different.

**To Reproduce**
Try same setting on LightGCN or most models with different stopping steps (50/30 etc.)

**Comment by enoche** [comment]
I extract parts of your code into a recommendation framework with Implicit feedback only.
Please refer to code for solving this bug:
https://github.com/enoche/ImRec

I think the bug occurs because of  `self.restore_user_e, self.restore_item_e`.

**Comment by hyp1231** [comment]
Hi, the mentioned bug has been solved in #897.

We noticed that ImRec inherits a large proportion of fundamental code from an old version RecBole. We are welcome any use and extension of RecBole. However, we strongly suggest that you explicitly mention that the code is inherited from RecBole.

Additional, by using RecBole in a manner of `import recbole`, e.g., [NCL](https://github.com/RUCAIBox/NCL), one can easily keep up-to-date to RecBole's bug fixing as well as efficiency improvements. Directly extracting source code of an old version may result unpredictable bugs like this.
